src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -432,7 +432,7 @@ Another common reason is that your email has been busted by spam filters. The fi
 
 ### Fine tune configuration for RabbitMQ
 
-In order to reduce delayed emails in mail queue, an adminstrator will adjust following properties in [rabbitmq.properties](https://github.com/apache/james-project/blob/master/src/site/xdoc/server/config-rabbitmq.xml) based on periodical demand.
+In order to reduce delayed emails in mail queue, an administrator will adjust the following properties in [rabbitmq.properties](https://github.com/apache/james-project/blob/master/src/site/xdoc/server/config-rabbitmq.xml) corresponding to situational conditions.
```

Comment:
In order to adapt mail queue settings to the actual traffic load, an administrator needs to perform fine configuration tunning as explained in [rabbitmq.properties](https://github.com/apache/james-project/blob/master/src/site/xdoc/server/config-rabbitmq.xml)

Reply:
OK thanks for your suggestion!